plugins/module_utils/foreman_helper.py

```diff
@@ -511,7 +524,8 @@ def connect(self):
             verify_ssl=self._foremanapi_validate_certs,
         )
 
-        self.status()
+        _status = self.status()
+        self.foreman_version = _status.get('version', '0.0.0')
```

Comment:
Should this already be a `LooseVersion` object?

Reply:
I debated that and wasn't sure where else I'd might need the string.

On the other hand, there are no other users of that right now, and `LooseVersion` has a working `__str__`… Yeah, I'll change that.